monai/apps/detection/transforms/array.py

```diff
@@ -296,3 +305,61 @@ def __call__(  # type: ignore
         """
 
         return flip_boxes(boxes, spatial_size=spatial_size, flip_axes=self.spatial_axis)
+
+
+class ClipBoxToImage(Transform):
```

Comment:
Please add the new transforms into `__all__` list:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/apps/detection/transforms/array.py#L28

Thanks.